Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -420,4 +447,8 @@ void CommandLineArgs::CheckForInvalidArguments()
     {
         throw hresult_invalid_argument(L"Cannot save tensor output if no input data is provided!");
     }
+    if (m_imagePaths.size() > 1 && IsSaveTensor())
+    {
+        throw hresult_not_implemented(L"Saving tensor output for multiple images isn't implemented yet.");
```

Comment:
I'm not sure if you should say implemented _yet_. It in a way commits you to the public to implement this in the future, and I don't think we should be putting those sorts of commitments in code comments. What do you think?

Reply:
I completely agree with your points.  Let me remove " _yet_ "